string duckie_name
uint8 location
string route
---
string result
